module/counters/monotonous_counter_test.go

```diff
@@ -16,6 +16,12 @@ func TestSet(t *testing.T) {
 	require.Equal(t, uint64(4), counter.Value())
 }
 
+func TestIncrement(t *testing.T) {
```

Comment:
can you also add a concurrent test that calls increment similar to the test below. The test should ensure that the values are sequential

Reply:
done